src/overlays/actors/ovl_En_Hanabi/z_en_hanabi.c

```diff
@@ -32,38 +33,328 @@ const ActorInit En_Hanabi_InitVars = {
     (ActorFunc)NULL,
 };
 
-#endif
+Vec3s D_80B23AA0[] = {
+    { 0x55, 0xE, 0x31 },    { 0x46, -0x46, 0 },     { 0x55, 0xE, -0x31 },    { 0x5A, -0x20, 0x1C },
+    { 0x5A, -0x20, -0x1C }, { 0x62, 0x10, 0 },      { 0x46, 0x46, 0 },       { 0x54, 0x2D, 0x1A },
+    { 0x54, 0x2D, -0x1A },  { 0x32, -0x32, 0x46 },  { 0x49, -0x13, 0x40 },   { 0x41, -0x41, 0x26 },
+    { 0x32, -0x32, -0x46 }, { 0x41, -0x41, -0x26 }, { 0x49, -0x13, -0x40 },  { 0, 0, -0x64 },
+    { 0xE, 0x55, -0x31 },   { 0x31, 8, -0x56 },     { 8, 0x31, -0x56 },      { 0x39, 0x39, -0x39 },
+    { 0x2D, 0x54, -0x1A },  { 0x1B, -0x1B, -0x5C }, { -0x32, 0x32, -0x46 },  { -0x1B, 0x1B, -0x5C },
+    { -0x13, 0x49, -0x40 }, { -0x46, 0x46, 0 },     { 0xE, 0x55, 0x31 },     { -0x20, 0x5A, -0x1C },
+    { -0x20, 0x5A, 0x1C },  { 0x10, 0x62, 0 },      { 0x2D, 0x54, 0x1A },    { -0x41, 0x41, -0x26 },
+    { -0x32, 0x32, 0x46 },  { -0x41, 0x41, 0x26 },  { -0x13, 0x49, 0x40 },   { 0, 0, 0x64 },
+    { 8, 0x31, 0x56 },      { 0x31, 8, 0x56 },      { 0x39, 0x39, 0x39 },    { -0x1B, 0x1B, 0x5C },
+    { 0x1B, -0x1B, 0x5C },  { -0xE, -0x55, -0x31 }, { -0xE, -0x55, 0x31 },   { 0x20, -0x5A, -0x1C },
+    { 0x20, -0x5A, 0x1C },  { -0x10, -0x62, 0 },    { -0x46, -0x46, 0 },     { -0x2D, -0x54, -0x1A },
+    { -0x2D, -0x54, 0x1A }, { 0x13, -0x49, -0x40 }, { 0x13, -0x49, 0x40 },   { -0x55, -0xE, -0x31 },
+    { -0x31, -8, -0x56 },   { -8, -0x31, -0x56 },   { -0x39, -0x39, -0x39 }, { -0x54, -0x2D, -0x1A },
+    { -0x49, 0x13, -0x40 }, { -0x55, -0xE, 0x31 },  { -0x5A, 0x20, 0x1C },   { -0x5A, 0x20, -0x1C },
+    { -0x62, -0x10, 0 },    { -0x54, -0x2D, 0x1A }, { -0x49, 0x13, 0x40 },   { -8, -0x31, 0x56 },
+    { -0x31, -8, 0x56 },    { -0x39, -0x39, 0x39 },
+};
```

Comment:
These seem to just be positions stored as shorts, I think they'll be better in dec.
```suggestion
Vec3s D_80B23AA0[] = {
    { 85, 14, 49 },    { 70, -70, 0 },     { 85, 14, -49 },    { 90, -32, 28 },
    { 90, -32, -28 }, { 98, 16, 0 },      { 70, 70, 0 },       { 84, 45, 26 },
    { 84, 45, -26 },  { 50, -50, 70 },  { 73, -19, 64 },   { 65, -65, 38 },
    { 50, -50, -70 }, { 65, -65, -38 }, { 73, -19, -64 },  { 0, 0, -100 },
    { 14, 85, -49 },   { 49, 8, -86 },     { 8, 49, -86 },      { 57, 57, -57 },
    { 45, 84, -26 },  { 27, -27, -92 }, { -50, 50, -70 },  { -27, 27, -92 },
    { -19, 73, -64 }, { -70, 70, 0 },     { 14, 85, 49 },     { -32, 90, -28 },
    { -32, 90, 28 },  { 16, 98, 0 },      { 45, 84, 26 },    { -65, 65, -38 },
    { -50, 50, 70 },  { -65, 65, 38 },  { -19, 73, 64 },   { 0, 0, 100 },
    { 8, 49, 86 },      { 49, 8, 86 },      { 57, 57, 57 },    { -27, 27, 92 },
    { 27, -27, 92 },  { -14, -85, -49 }, { -14, -85, 49 },   { 32, -90, -28 },
    { 32, -90, 28 },  { -16, -98, 0 },    { -70, -70, 0 },     { -45, -84, -26 },
    { -45, -84, 26 }, { 19, -73, -64 }, { 19, -73, 64 },   { -85, -14, -49 },
    { -49, -8, -86 },   { -8, -49, -86 },   { -57, -57, -57 }, { -84, -45, -26 },
    { -73, 19, -64 }, { -85, -14, 49 },  { -90, 32, 28 },   { -90, 32, -28 },
    { -98, -16, 0 },    { -84, -45, 26 }, { -73, 19, 64 },   { -8, -49, 86 },
    { -49, -8, 86 },    { -57, -57, 57 },
};
```